src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/MIReplicaReportingServiceTest.java

```diff
@@ -550,6 +677,22 @@ void should_report_incomplete_task_history(String initialState, String lastActio
 
     }
 
+    private TaskResource buildTaskResource() {
+        TaskResource taskResource = new TaskResource(
+            UUID.randomUUID().toString(),
+            "someTaskName",
+            "someTaskType",
+            UNASSIGNED,
+            "987654",
+            OffsetDateTime.parse("2022-05-09T20:15:45.345875+01:00")
+        );
+        taskResource.setCreated(OffsetDateTime.parse("2022-05-05T20:15:45.345875+01:00"));
+        taskResource.setPriorityDate(OffsetDateTime.parse("2022-05-15T20:15:45.345875+01:00"));
+        taskResource.setLastUpdatedTimestamp(OffsetDateTime.parse("2022-05-05T20:15:45.345875+01:00"));
+        taskResource.setLastUpdatedAction("Configure");
+        return taskResource;
```

Comment:
This is a very inflexible method that's difficult to re-use in a different context.  The next dev can't really use it if for instance they wanted a slightly different createdDate.